QA_Requests/MultiSelectPerf/import_data_using_api.py

```diff
@@ -0,0 +1,40 @@
+import configparser
+import requests
+
+
+def post_api(URL, input_payload, login_user, login_pass, header, type="json"):
```

Comment:
even this file should be in the testPage directory or some other directory like testMethods.

Reply:
Again this would move to a common utility since it is being used in https://github.com/dimagi/dimagi-qa/pull/137 and would be used in perf tickets going forward.